azurerm/cmd/generate.go

```diff
@@ -1,5 +1,6 @@
 package main
 
+//This file generates list methods for the defined resources in azurerm/reader_generated.go file using the templates defined in template.go
```

Comment:
Add the `\s` beteween the `//` and the first word.

IDK if this comment is "useful". The "file" does not generate anything, what does is the `func main(){}` on this file or the `func generate(){}` to be more precise, so maybe this would fit better in one of those.

Or you could add a `doc.go` with a comment above the `package main` which would generate a nice `go doc` of the full package, as at the end this what the full package is doing.

An example is https://github.com/cycloidio/terracognita/blob/master/filter/doc.go which then generates this https://pkg.go.dev/github.com/cycloidio/terracognita@v0.7.1/filter